wagtail/views.py

```diff
@@ -35,9 +35,8 @@ def authenticate_with_password(request, page_view_restriction_id, page_id):
     """
     restriction = get_object_or_404(PageViewRestriction, id=page_view_restriction_id)
     page = get_object_or_404(Page, id=page_id).specific
-
+    form = PasswordViewRestrictionForm(instance=restriction, request.POST or None)
```

Comment:
This change makes the code more difficult to follow. The bound form (with `request.POST` passed) and the unbound one serve different purposes, so it's appropriate to keep the two versions separate.